modules/apps/static/portal-osgi-web/portal-osgi-web-wab-generator/src/main/java/com/liferay/portal/osgi/web/wab/generator/internal/processor/WabProcessor.java

```diff
@@ -448,6 +448,7 @@ protected void processDeclarativeReferences(Analyzer analyzer)
 
 		processDefaultServletPackages();
 		processTLDDependencies(analyzer);
+		processPortalListenerClassesDependencies(analyzer);
```

Comment:
@ericyanLr  can you format it like so:
```
...
		processDefaultServletPackages();
 		processTLDDependencies(analyzer);
		// blank line here
		processPortalListenerClassesDependencies(analyzer);
		// blank line here
		Path pluginPath = _pluginDir.toPath();
...
```

Reply:
@rotty3000 Will do, thanks for the clarification!